roles/nvidia-network-operator/tasks/k8smaster.yaml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
Can we give this file a more descriptive (install-network-operator.yml) or more standard (main.yml) name for this file?